app/javascript/articles/components/CommentsList.jsx

```diff
@@ -36,7 +36,12 @@ export const CommentsList = ({ comments = [], articlePath, totalCount }) => {
     return (
       <div className="crayons-story__comments">
         {comments.slice(0, numberOfCommentsToShow).map((comment) => {
-          return <CommentListItem comment={comment} />;
+          return (
+            <CommentListItem
+              key={`${comment.published_at_int}-${comment.name}`}
```

Comment:
Refactor (blocking): Use the comment ID property available in the comment object. It is guaranteed to be unique.